app/komponenter/samtykkepanel/SamtykkePanel.tsx

```diff
@@ -34,13 +34,17 @@ const SamtykkePanel: React.FC<Props> = ({
         onChange={() => {
           vedSamtykkeEndring(!erSamtykkeBekreftet);
         }}
+        data-testid="samtykkepanel"
         error={
           !erSamtykkeBekreftet &&
           feilmeldingAktivert && (
-            <TekstBlokk
-              tekstblokk={tekster.samtykkePanelFeilmelding}
-              typografi={ETypografiTyper.BODY_SHORT}
-            /> //denne gir feilmelding fordi den ikke er en ren String ("kan ikke være i <p>"). Visuelt fungerer den.
+            <span data-testid="samtykkepanelFeilmelding">
+              <TekstBlokk
+                tekstblokk={tekster.samtykkePanelFeilmelding}
+                typografi={ETypografiTyper.BODY_SHORT}
+              />
+              {/*denne gir feilmelding fordi den ikke er en ren String ("kan ikke være i <p>"). Visuelt fungerer den.*/}
+            </span>
```

Comment:
Må ting wrappes i en span for å kunne teste det som ligger inni? 
Og feiler testen her (ref kommentaren)?

Reply:
Kommentaren i koden handlet om at en p var inni en p og er løst i en annen PR [her](https://github.com/bekk/nav-familie-endringsmelding/pull/70), så hadde ikke noe med testene å gjøre.  
Smart å legge til prop som nevnt under! Det er gjort [her](https://github.com/bekk/nav-familie-endringsmelding/pull/63/commits/481decdcd64b887c3ba1b0c31011d8f6d91f1eb7)